migrate-test

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
I'm not keen on scripts being added to the project root as that's generally a space for dependency configurations and is very cluttered. Arguably, these could live in `scripts` although they're not intended to be run directly so perhaps some other folder would be best? If the team is generally happy with scripts in the root, I'll put up with it.

Reply:
The script is located by the files that uses it - docker, docker-compose, etc.

Perhaps we could put all our docker and compose files in a sub folder, but I don't see that very much in other repos. Docker etc tend to be at the root.

It's a larger question of folder structure, which hasn't been addressed in our standards at https://eaflood.atlassian.net/wiki/spaces/FFCPD/pages/1297219944/Standards+for+Development+Project+Structure+and+Naming+Conventions

Reply:
That's reasonable. I have seen docker-compose files all in a subfolder in a few repos before. That might be a good option for us as we don't want to run `docker-compose` commands directly from the root (we should always use the `start` script to ensure migrations are run before the app container starts).

I'm happy to discuss this separately and leave as-is for this PR.